crypto/s2n_cipher.h

```diff
@@ -58,12 +58,24 @@ struct s2n_aead_cipher {
     int (*encrypt) (struct s2n_session_key *key, struct s2n_blob *iv, struct s2n_blob *add, struct s2n_blob *in, struct s2n_blob *out);
 };
 
+struct s2n_comp_cipher {
```

Comment:
Any reason not to make comp just be "composite"? it's a little longer, but more clear.